src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -1326,7 +1326,7 @@ def _create_db_wait_for_first_backup(test, resource_group, server, database_name
     # Wait until earliestRestoreDate is in the past. When run live, this will take at least
     # 10 minutes. Unforunately there's no way to speed this up
     while db['earliestRestoreDate'] is None:
-        sleep(60)
+        time.sleep(60)
```

Comment:
`from time import sleep` can now be removed.

Reply:
We need modify these codes before remove `from time import sleep`.
I think add `if self.is_live` condition is also acceptable.
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L1125-L1126
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L1454-L1456
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L2743-L2744
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L2764-L2765
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L2777-L2778
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L4481-L4482
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L4594-L4595
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L4607-L4608

Reply:
Let's only remove `from time import sleep` when it is not used.